### ISSUE TITLE
Hide registration button if public registration is disabled

### ISSUE THREAD

**Issue by deshack** [SqueezyWeb/KISS#5]
- [x] This is a feature request
- [ ] This is a bug report
- [x] This request isn't a duplicate of an [existing issue](https://github.com/SqueezyWeb/KYSS/issues)

---

**What is the current behavior?**

Registration is public to all and registration button is always visible to non-authenticated users.

**What is the expected or desired behavior?**

Registration button is visible to non-authenticated users only if the application settings allow for public registration.

---
## Feature Request

**Please provide use cases for changing the current behavior:**

X

**Other relevant information:**

Is blocked by #4
